Cargo.toml

```diff
@@ -191,6 +191,317 @@ resolver = "2"
 [workspace.package]
 rust-version = "1.83.0"
 
+[workspace.dependencies]
```

Comment:
Seeing all of this in one place really makes you realize just how much external code we are explicitly dependent on... (and this includes all other dependencies that these crates depend on)